os/kernel/task_monitor/task_monitor.c

```diff
@@ -37,14 +84,40 @@ void task_monitor_update_list(int pid, int status)
 int task_monitor(int argc, char *argv[])
 {
 	int pid_idx;
+	int time_idx;
+	int max_time;
 	struct tcb_s *tcb;
+	taks_monitor_t *next_mon;
+
+	max_time = CONFIG_TASK_MONITOR_MAX_INTERVAL / CONFIG_TASK_MONITOR_INTERVAL;
+
+	/* Initialize the dequeue list */
```

Comment:
from line 94 to 110 are all initialization.  How about making static function for initializing?

Reply:
done.